app/assets/stylesheets/govuk_publishing_components/components/_metadata.scss

```diff
@@ -0,0 +1,70 @@
+.gem-c-metadata {
+  @include core-16;
+  @include responsive-bottom-margin;
+  @extend %contain-floats;
+
+  &.direction-rtl {
+    direction: rtl;
+    text-align: start;
+  }
+
+  dt {
+    margin-top: 0.5em;
+    line-height: normal;
+
+    @include media(tablet) {
+      float: left;
+      clear: left;
+      width: 30%;
+      box-sizing: border-box;
```

Comment:
This should use the box sizing mixin - https://github.com/alphagov/govuk_publishing_components/blob/9a338ca0bade6af4271e9e5d8d0b441225f9f900/app/assets/stylesheets/govuk_publishing_components/components/_share-links.scss#L9

Reply:
Where does this mixin come from?

Reply:
https://github.com/alphagov/govuk_frontend_toolkit/blob/452f82e7538d7f5b8f73ee7d81abea9bc857af13/stylesheets/_css3.scss#L68

Reply:
Is it necessary? I've found 8 instances of `box-sizing: border-box` in other CSS files.

Reply:
I think we should be using the mixin if only to ensure consistency. We should probably get around to updating those other 8 instances as well at some point.

Reply:
There are more instances of `box-sizing: border-box` than `@include box-sizing`, so for more consistency we would keep `box-sizing: border-box`. I guess I'm not sure why we need the browser prefixes if we've been doing fine without? (I've done it anyway to move it along).